common/decisions/dvg_macedon.txt

```diff
@@ -0,0 +1,46 @@
+﻿dvg_mcd_acceptgreeks = {
+	is_shown = {
+		exists = c:YUG
+		root = c:YUG
+		NOT = {
+			c:YUG = {
+				country_has_primary_culture = cu:greek
+			}
+		}
+		c:YUG = {
+			was_formed_from = c:MCD

```

Comment:
PostValidate of trigger 'was_formed_from' returned false at  file: common/decisions/dvg_macedon.txt line: 11

Decisions is also not showing up after forming YUG as MCD